ghcide/test/exe/Main.hs

```diff
@@ -1520,6 +1520,36 @@ extendImportTests = testGroup "extend import actions"
                     , "import ModuleA as A (stuffB, (.*))"
                     , "main = print (stuffB .* stuffB)"
                     ])
+        , testSession "extend single line import with infix constructor" $ template
```

Comment:
a nice way to merge it before the fix is annotate the tests with a knownBroken for a specific ghc version, like other tests in the file

Reply:
for example 
https://github.com/haskell/haskell-language-server/blob/016ff739c4be0dcb50e7a8575a4e8fefe3a58d5a/ghcide/test/exe/Main.hs#L2639

Reply:
Done, thank you.